Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Build compilation image
+FROM ruby:2.7.2-alpine as builder
+
+# The application runs from /app
+WORKDIR /app
+
+# Add the timezone as it's not configured by default in Alpine
+RUN apk add --update --no-cache tzdata && \
+  cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
+  echo "Europe/London" > /etc/timezone
+
+# build-base: complication tools for bundle
+# yarn: node package manager
+# postgresql-dev: postgres driver and libraries
+RUN apk add --no-cache build-base yarn postgresql-dev
+
+# Install bundler to run bundle exec
+# This should be the same version as the Gemfile.lock
+RUN gem install bundler:2.1.4 --no-document
+RUN bundle config set without 'development test'
```

Comment:
`without test` ?
will gems required for running tests be installed?

Reply:
This image will be used to deploy to paas, no need for test gems. If they are required later on we can create a story to create a separate image